config_templates/jobs.json

```diff
@@ -7,6 +7,7 @@
                 "abc": "xyz"
             },
             "environment_variables_secret_id": "https://myvault.vault.azure.net/secrets/myjobenv",
+            "max_task_retry_count": 1,
```

Comment:
Happy to do this; I originally wanted to be consistent with the Batch SDK for python etc, since they use `max_task_retry_count`